src/plugins/saved_objects_management/public/management_section/objects_table/components/import_summary.tsx

```diff
@@ -194,11 +194,18 @@ export const ImportSummary = ({ failedImports, successfulImports }: ImportSummar
         }
       >
         <h3>
-          <FormattedMessage
-            id="savedObjectsManagement.importSummary.headerLabel"
-            defaultMessage="{importCount} object(s) imported"
-            values={{ importCount: importItems.length }}
-          />
+          {importItems.length === 1 ? (
+            <FormattedMessage
+              id="savedObjectsManagement.importSummary.headerLabelSingular"
+              defaultMessage="1 object imported"
+            />
+          ) : (
+            <FormattedMessage
+              id="savedObjectsManagement.importSummary.headerLabelPlural"
+              defaultMessage="{importCount} objects imported"
+              values={{ importCount: importItems.length }}
+            />
+          )}
```

Comment:
FYI (don't bother changing): The `defaultMessage` syntax supports "pluralizing" without having to conditionally render in code like this. The syntax for this is rather wonky in my opinion though, so what you have here is likely clearer:

https://github.com/elastic/kibana/blob/9aad8986e1f11b199af1ce0a1570a1c54995ade4/src/plugins/es_ui_shared/static/forms/helpers/field_validators/index_name.ts#L56-L63

Reply:
TIL!